libpod/rootless_cni_linux.go

```diff
@@ -25,7 +25,8 @@ import (
 
 // Built from ../contrib/rootless-cni-infra.
 var rootlessCNIInfraImage = map[string]string{
-	"amd64": "quay.io/libpod/rootless-cni-infra@sha256:8aa681c4c08dee3ec5d46ff592fddd0259a35626717006d6b77ee786b1d02967", // 1-amd64
+	// TODO: move back to quay
+	"amd64": "ghcr.io/akihirosuda/podman-rootless-cni-infra@sha256:b5687cc3f76d7b98902c1ce09998dac57d629b3324b99462853183ac059bdf36", // 2-g7bffd49ad-amd64
```

Comment:
@vrothberg Could you push the image to quay?

Reply:
Sure! I'll push it in a minute. Do you have an account on Quay? I can give you write permission for the specific namespace.

Reply:
Sorry I don't have

Reply:
quay.io/libpod/rootless-cni-infra:2-amd64 is uploaded (sha256:e92c3a6367f8e554121b96d39af1f19f0f9ac5a32922b290112e13bc661d3a29)

Reply:
Updated the image, thanks @vrothberg